utilities/README.md

```diff
@@ -0,0 +1,24 @@
+# Healthcare Toolkit
+
+The healthcare toolkit is designed to guide various data movement patterns including data ingestion, consumption, and auditing. The utilities provide data movement patterns between various sources/sinks and Google Cloud healthcare products. 
+Healthcare Toolkit Utilities.
+
+## Ingest
+1. [Kafka to FHIR Store](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/ingest/dataflow_kafka_to_fhirstore) - Consume messages via Kafka, and write to a GCP FHIR Store.
+2. [Pub/Sub to FHIR Store](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/ingest/dataflow_pubsub_to_fhirstore) - Consume messages via Pub/Sub, and write to a GCP FHIR Store.
+
+## Consume
+1. [FHIR Store to GCS](https://github.com/GoogleCloudPlatform/healthcare/tree/master/utilities/consume/dataflow_fhirstore_bq_consume) - Consumes a notification from a GCP FHIR Store, and writes the message to GCS.
```

Comment:
Consumes a notification containing FHIR resource in JSON format along with attributes from Google Cloud Healthcare FHIR Store, reads the latest messages from BigQuery, transforms it and writes the message to GCS.

Reply:
@surjits254 Feel free to add your comments this utility.

Reply:
lgtm, what you are suggesting

Reply:
@surjits254 nothing as you have worked on this one, tagged you incase you want to add more details.

Reply:
Yes, I am good with changes you are suggesting!!